### PR TITLE
Use different argument for cassandra address in Admin CLI

### PR DESCRIPTION
**Pull request by longquanzheng**
https://github.com/uber/cadence/issues/2663

### CODE REVIEW

File: tools/cli/adminCommands.go [hunk 1]
```diff
@@ -263,7 +263,7 @@ func readOneRow(query *gocql.Query) (map[string]interface{}, error) {
 }
 
 func connectToCassandra(c *cli.Context) *gocql.Session {
-	host := getRequiredOption(c, FlagAddress)
+	host := getRequiredOption(c, FlagCassandraAddress)
```
Comment (wxing1292) [inline tools/cli/adminCommands.go]: could you also add a `FlagCassandraPort`?

Comment (wxing1292) [reply tools/cli/adminCommands.go]: line 270 still use the `FlagPort `

Comment (longquanzheng) [reply tools/cli/adminCommands.go]: done

**Comment by longquanzheng** [comment]
related to https://github.com/uber/cadence/issues/2777

**Pull request merged**